docs/Releases/v0.2.0-release.md

```diff
@@ -0,0 +1,364 @@
+NAME              AGE
+argocd-operator   53s
+
+$  kubectl get operator argocd-operator -o yaml 
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"operators.operatorframework.io/v1alpha1","kind":"Operator","metadata":{"annotations":{},"name":"argocd-operator"},"spec":{"packageName":"argocd-operator"}}
+  creationTimestamp: "2023-06-21T14:57:50Z"
+  generation: 1
+  name: argocd-operator
+  resourceVersion: "10690"
+  uid: 6e0c67a5-eb9c-41c6-a455-140b28714d34
+spec:
+  packageName: argocd-operator
+status:
+  conditions:
+  - lastTransitionTime: "2023-06-21T14:57:51Z"
+    message: resolved to "quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6"
+    observedGeneration: 1
+    reason: Success
+    status: "True"
+    type: Resolved
+  - lastTransitionTime: "2023-06-21T14:57:57Z"
+    message: installed from "quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6"
+    observedGeneration: 1
+    reason: Success
+    status: "True"
+    type: Installed
+  installedBundleResource: quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6
+  resolvedBundleResource: quay.io/operatorhubio/argocd-operator@sha256:1a9b3c8072f2d7f4d6528fa32905634d97b7b4c239ef9887e3fb821ff033fef6
+```
+
+Looks like the operator has been installed successfully. We can confirm this by looking at the workloads that were created as a result of this operator installation: 
+
+```bash 
+$ kubectl get namespaces | grep argocd 
+argocd-operator-system       Active   4m17s
+
+$ kubectl get pods -n argocd-operator-system 
+NAME                                                 READY   STATUS    RESTARTS   AGE
+argocd-operator-controller-manager-bb496c545-ljbbr   2/2     Running   0          4m32s
+$ kubectl get crds | grep argo 
+NAME                                           CREATED AT
+applications.argoproj.io                       2023-06-21T15:33:01Z
+applicationsets.argoproj.io                    2023-06-21T15:33:01Z
+appprojects.argoproj.io                        2023-06-21T15:33:01Z
+argocdexports.argoproj.io                      2023-06-21T15:33:01Z
+argocds.argoproj.io                            2023-06-21T15:33:01Z
+```
+
+To delete the operator, simply delete the Operator CR that was created earlier: 
+
+```bash
+$ kubectl delete operator argocd-operator 
+operator.operators.operatorframework.io "argocd-operator" deleted
+$ kubectl get namespaces | grep argocd
+$ 
+$ kubectl get crds | grep argo 
+$
+```
+
+**Notice again a key difference between OLM v0 and this v0.2.0 release of OLM when deleting an operator: besides the fact that deleting an installed operator is as easy as deleting the Operator CR, it also deletes every workload that was created during installation, including the CRDs**
```

Comment:
This is more along the lines of what I was thinking for a "note" about the OLMv1 differences instead of calling them out as we are going through steps. I think visually it would be nice to try and make use of the [just-the-docs callouts UI](https://just-the-docs.com/docs/ui-components/callouts/) rather than just bold text